packages/trader/src/AppV2/Components/OrderDetails/order-details.tsx

```diff
@@ -25,11 +25,21 @@ const OrderDetails = ({ contract_info }: ContractInfoProps) => {
                             </div>
                             <div className='order-details__table-row-cell'>
                                 {Array.isArray(value) ? (
-                                    value.map((val, index) => (
-                                        <Text key={`${key}_${index}`} size='sm'>
-                                            {val}
-                                        </Text>
-                                    ))
+                                    value.map((val, index) =>
+                                        ['number', 'string'].includes(typeof val) ? (
+                                            <Text key={`${key}_${index}`} size='sm'>
+                                                {val}
```

Comment:
Is val or val.caption unique? Maybe it will be possible to use them instead as keys. Please, ignore if they are not unique

Reply:
It is from the array , we can have value in two or more lines with same style as well as with caption style . So when we need caption style , we can pass it as an object with caption key value .